driver-core/src/main/com/mongodb/event/CommandEvent.java

```diff
@@ -39,13 +59,12 @@ public abstract class CommandEvent {
      * @param connectionDescription the connection description
      * @param commandName the command name
      * @since 4.4
+     * @deprecated Prefer {@link CommandEvent#CommandEvent(RequestContext, long, int, ConnectionDescription, String)}
      */
+    @Deprecated
```

Comment:
Kind of annoying that we have to preserve BC on constructors like this but it's what we do.